backend/src/main/kotlin/app/ehrenamtskarte/backend/mail/Mailer.kt

```diff
@@ -212,6 +212,36 @@ object Mailer {
         )
     }
 
+    fun sendApplicationMailToContactPerson(
+        backendConfig: BackendConfiguration,
+        projectConfig: ProjectConfig,
+        contactPerson: String,
+        personalData: PersonalData,
+        accessKey: String
+    ) {
+        val subject = "Antrag erfolgreich eingereicht"
+        val message = emailBody {
+            p { +"Sehr geehrte/r $contactPerson," }
+            p { +"Ihr Antrag auf die Bayerische Ehrenamtskarte für ${personalData.forenames.shortText} ${personalData.surname.shortText} wurde erfolgreich eingereicht." }
+            p {
+                +"Den aktuellen Status Ihres Antrags können sie jederzeit unter folgendem Link einsehen. Dort haben Sie auch die Möglichkeit, Ihren Antrag bei Bedarf zurückzuziehen:"
+                br()
+                link(URL("${projectConfig.administrationBaseUrl}/antrag-einsehen/${urlEncode(accessKey)}"))
+            }
+            p { +"Bei Rückfragen wenden Sie sich bitte direkt an Ihr zuständiges Landratsamt oder die Verwaltung Ihrer kreisfreien Stadt." }
+            p { +DO_NOT_ANSWER_MESSAGE }
+            p { +"- ${projectConfig.administrationName}" }
```

Comment:
"Mit freundlichen Grüßen" is missing (was in the toni's suggestion)
not sure if it must be included

Reply:
alright good find. but then we should add this to all mails